### PR TITLE
🦹  Export super-mechanisms from `DensityNernst`

### PR DESCRIPTION
**Pull request by thorstenhater**
Add export for ion channels added via `ChannelDensityNernst`

@llandsmeer could you add similar functionality for the `NonUniform*` 
variants? It should mesh with the current setup, you just need to add
the non-uniform parameters to the non-flatten list.

### CODE REVIEW

**Comment by llandsmeer** [comment]
The NonUniform* ones will be a separate PR right?

**Comment by thorstenhater** [comment]
Yes, small steps.

File: src/acc.rs [hunk 1]
```diff
@@ -498,20 +450,56 @@ pub fn biophys(
     Ok(decor)
 }
 
-#[allow(non_snake_case)] // xml..
+fn make_variable_parameter_map(
```
Comment (llandsmeer) [inline src/acc.rs:453]: And all of this as well

**Review by llandsmeer — APPROVED** [review]
I'm approving - but I don't understand how this PR contains all the changes from the previous PR?

**Comment by thorstenhater** [comment]
That's since I forked it off the previous feature branch, not `main`. The diff is the same, tough.

**Pull request merged**